spec/features/inspection_display_spec.rb

```diff
@@ -37,6 +37,15 @@
     end
   end
 
+  context "on item index" do
+    let(:an_item) { create(:item, item_type_id: 1) }
+    let!(:an_inspection) { create(:inspection, item: an_item) }
```

Comment:
Since you only create one inspection, there is no assurance that you're testing getting the most recent inspection.